cores/esp8266/LwipIntfDev.h

```diff
@@ -386,9 +392,9 @@ err_t LwipIntfDev<RawDev>::handlePackets()
 }
 
 template <class RawDev>
-void LwipIntfDev<RawDev>::setDefault()
+void LwipIntfDev<RawDev>::setDefault(bool deflt)
 {
-    _default = true;
+    _default = deflt;
```

Comment:
`if (_default && connected())`?

Reply:
hm. not really resolved, since we always apply netif_set_default regardless of value
maybe just remove the below and leave it to the initialization method(s)?

Reply:
logic is updated